service/src/main/java/org/apache/celeborn/server/common/service/model/ClusterInfo.java

```diff
@@ -17,16 +17,16 @@
 
 package org.apache.celeborn.server.common.service.model;
 
-import java.util.Date;
+import java.time.Instant;
 
 public class ClusterInfo {
 
   private Integer id;
   private String name;
   private String namespace;
   private String endpoint;
-  private Date gmtCreate;
-  private Date gmtModify;
+  private Instant gmtCreate;
```

Comment:
Are there compatibility issues when converting a timestamp to an Instant and vice versa using iBatis?

Reply:
@RexXiong, I have added the verification in https://github.com/apache/celeborn/pull/2555/commits/6e9740c0663644e35c1b86fe7271792a7cdfd935. PTAL.